### DISCUSSION TITLE
Does cross support no_std build (embedded)

### DISCUSSION THREAD

**Discussion by baoyachi** [cross-rs/cross#966] [General]

**Comment by Alexhuszagh** [comment]
Yep, it does. Just the system running the cross-compiler needs `std`, for obvious reasons. We support numerous `no_std` targets, notably the `thumb*` targets. The targets we currently support for `no_std` are:

- `thumbv6m-none-eabi`
- `thumbv7em-none-eabi`
- `thumbv7em-none-eabihf`
- `thumbv7m-none-eabi`

But feel free to add more targets, or request them. We have [cross-toolchains](https://github.com/cross-rs/cross-toolchains) for lower tier targets we don't build in CI but support.

**Reply by Alexhuszagh** [reply]
Also, the most likely targets we'd add afterwards are the RISC-V and ARMv8 bare-metal targets, since they're both Tier 2 and therefore have much better support.
